tagging/apps/apply_tags.py

```diff
@@ -15,9 +15,6 @@
 
 
 def apply_tags(config: Config) -> None:
-    """
```

Comment:
These docstrings are not helpful (especially in apps), and not consistent at all